static_report/src/components/ComparisonReport/CRTableList/CRTableListColumnsSummary.tsx

```diff
@@ -0,0 +1,38 @@
+import { Flex, Text, Icon } from '@chakra-ui/react';
+import {
+  FiArrowUpCircle,
+  FiArrowDownCircle,
+  FiArrowRight,
+} from 'react-icons/fi';
+
+export function CRTableListColumnsSummary({
+  base,
```

Comment:
what kind of `number` are these Props? 
e.g. `baseTotal`, `baseCount` etc

Reply:
done